views/partials/header.html

```diff
@@ -1,89 +1,114 @@
+                    {{#if component.isComponent}}Demos{{else}}Info{{/if}}
+                    </a>
+                </li>
+                <li>
+                    <a href="/components/{{ component.name }}@{{ component.version }}/readme{{#if @root.currentBrand}}?brand={{@root.currentBrand}}{{/if}}"
+                       {{#ifEquals nav 'readme'}}aria-current="true"{{/ifEquals}}>
+                    Readme
+                    </a>
+                </li>
+                {{#if component.showSassDoc}}
+                <li>
+                    <a href="/components/{{ component.name }}@{{ component.version }}/sassdoc{{#if @root.currentBrand}}?brand={{@root.currentBrand}}{{/if}}"
+                       {{#ifEquals nav 'sassdoc'}}aria-current="true"{{/ifEquals}}>
+                    SassDoc
+                    </a>
+                </li>
+                {{/if}}
+                {{#if component.showJSDoc}}
+                <li>
+                    <a href="/components/{{ component.name }}@{{ component.version }}/jsdoc{{#if @root.currentBrand}}?brand={{@root.currentBrand}}{{/if}}"
+                       {{#ifEquals nav 'jsdoc'}}aria-current="true"{{/ifEquals}}>
+                    JSDoc
+                    </a>
+                </li>
+                {{/if}}
+                {{#if component.showRequiredPolyfills}}
+                <li>
+                    <a href="/components/{{ component.name }}@{{ component.version }}/polyfills{{#if @root.currentBrand}}?brand={{@root.currentBrand}}{{/if}}"
+                       {{#ifEquals nav 'polyfills'}}aria-current="true"{{/ifEquals}}>
+                    Required Polyfills
+                    </a>
+                </li>
+                {{/if}}
+            </ul>
+        </div>
+        <button class="o-header-services__scroll-button o-header-services__scroll-button--left" title="scroll left"
+                aria-hidden="true" disabled></button>
+        <button class="o-header-services__scroll-button o-header-services__scroll-button--right" title="scroll right"
+                aria-hidden="true" disabled></button>
+    </nav>
+    {{/if}}
 
 </header>
 
-
+<div id="decomission-warning" class="o-message o-message--notice o-message--warning" data-o-component="o-message"
+     data-o-message-close="false">
+    <div class="o-message__container">
+        <div class="o-message__content">
+            <p class="o-message__content-main">
+                <span class="o-message__content-highlight">The Origami Registry is being decommissioned</span>
+                <span>by the end of March 2024 and replaced with <a href="storybook/">Storybook</a>. For more information please read our <a
+                        href="https://origami.ft.com/blog/2024/02/12/newsletter/#goodbye-registry">blog post</a>.</span>
+            </p>
+            <p class="o-message__content-additional">Are demos missing from Storybook? Let us know in
+                #origami-support.</p>
```

Comment:
How about this instead? I think we can be confident you've caught all the demos but people may have other concerns if they're unfamiliar with Storybook, or find their workflow disrupted in some way.

```suggestion
            <p class="o-message__content-additional">Please share any feedback in our #origami-chat Slack channel.</p>
```